libs/SalesforceSDK/src/com/salesforce/androidsdk/ui/components/LoginView.kt

```diff
@@ -166,23 +167,35 @@ fun LoginView() {
         },
         bottomBar = {
             BottomAppBar(containerColor = viewModel.dynamicBackgroundColor.value) {
-                if (viewModel.isIDPLoginFlowEnabled.value) {
-                    Button(
-                        onClick = { activity.onIDPLoginClick() },
-                        modifier = Modifier
-                            .fillMaxWidth()
-                            .height(50.dp)
-                            .padding(start = 20.dp, end = 20.dp),
-                        shape = (RoundedCornerShape(5.dp)),
-                        colors = ButtonDefaults.buttonColors(
-                            containerColor = colorResource(id = sf__primary_color),
-                            contentColor = colorResource(id = sf__secondary_color)
-                        )
-                    ) {
-                        Text(
-                            text = stringResource(id = sf__launch_idp),
-                            fontSize = 14.sp
-                        )
+                Row(
+                    modifier = Modifier.fillMaxWidth(),
+                    horizontalArrangement = Arrangement.Center,
+                ) {
+                    if (viewModel.isIDPLoginFlowEnabled.value) {
+                        Button(
+                            onClick = { activity.onIDPLoginClick() },
+                            modifier = Modifier
+                                .fillMaxWidth()
+                                .height(50.dp)
+                                .padding(start = 20.dp, end = 20.dp),
+                            shape = (RoundedCornerShape(5.dp)),
+                            colors = ButtonDefaults.buttonColors(
+                                containerColor = colorResource(id = sf__primary_color),
+                                contentColor = colorResource(id = sf__secondary_color)
+                            )
+                        ) {
+                            Text(
+                                text = stringResource(id = sf__launch_idp),
+                                fontSize = 14.sp
+                            )
+                        }
+                    }
+                    viewModel.additionalBottomBarButtons.value.forEach { button ->
```

Comment:
We probably should add a little todo here about adding back the bio auth login button.
Brandon had a comment about it - see [here](https://github.com/forcedotcom/SalesforceMobileSDK-Android/blame/7d7892fdc0d22658b618df2bf88e0f0db74d4eeb/libs/SalesforceSDK/src/com/salesforce/androidsdk/ui/components/LoginView.kt#L158) - it got lost in my last PR.